drivers/dma/CMakeLists.txt

```diff
@@ -1,5 +1,10 @@
 # SPDX-License-Identifier: Apache-2.0
 
+zephyr_syscall_header_ifdef(
```

Comment:
This header doesn't need to be added to the syscall headers

Reply:
Done